Lib/test/support/__init__.py

```diff
@@ -1068,7 +1068,7 @@ def change_cwd(path, quiet=False):
     """
     saved_dir = os.getcwd()
     try:
-        os.chdir(path)
+        os.chdir(os.path.realpath(path))
```

Comment:
Why is this needed?

Reply:
@serhiy-storchaka This ensures that the path is fully normalised, in particular, converting shortened paths to long paths. It has an impact during tests later on if the current username is longer than 8 characters.

But I think this change has been merged elsewhere already. When I rebase, it'll probably go away.